ttnn/cpp/ttnn/operations/moreh/moreh_helper_functions.hpp

```diff
@@ -0,0 +1,286 @@
+/*
+ * SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <optional>
+#include <variant>
+
+#include "ttnn/tensor/tensor.hpp"
+#include "tt_metal/host_api.hpp"
+
+namespace tt {
+namespace operations {
+namespace primary {
```

Comment:
```suggestion
namespace ttnn {
namespace operations {
```
The reason the namespace of moreh_helper was `primary` is that, in the past, all moreh ops used the `primary` namespace.
Since the moreh ops are no longer in the `primary` namespace, moving them to `ttnn::operations` would allow us to reduce unnecessary namespace code, as shown below
```C++
(std::uint32_t)tt::operations::primary::is_dram(src0_buffer) 
-> 
(std::uint32_t)(is_dram(src0_buffer))
```

However, if changing the namespace causes a lot of unexpected additional work, it’s not necessary to make this change in this PR